### PR TITLE
Fix for missing pages data struct in bdo2016 reader

### LINKED ISSUE #379: pymchelper bdo 2016 reader broken?
Seems to happen with detector in old format?

```
shieldhit/test/25_water_density$ convertmc image "ref_fl_zmsh.bdo"
Traceback (most recent call last):
  File "/home/bassler/.local/bin/convertmc", line 8, in <module>
    sys.exit(main())
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/run.py", line 153, in main
    status = convertfromlist(parsed_args.input,
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/input_output.py", line 176, in convertfromlist
    estimator = fromfilelist(filelist, error, nan)
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/input_output.py", line 77, in fromfilelist
    result = fromfile(input_file_list[0])
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/input_output.py", line 55, in fromfile
    if not reader.read(estimator):  # unsuccefful read
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/readers/common.py", line 23, in read
    result = self.read_data(estimator)
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/readers/shieldhit/reader_bdo2016.py", line 209, in read_data
    logger.debug("Detector data : " + str(estimator.data))
AttributeError: 'Estimator' object has no attribute 'data'

shieldhit/test/25_water_density$ convertmc --version
1.3.0
```
bdo file from 2017.
Screenshot_2021-03-13_10-58-30

also recently generated files:
```
bassler@kepler:~/Projects/shieldhit/test/25_water_density$ convertmc image "fl_zmsh.bdo"
Traceback (most recent call last):
  File "/home/bassler/.local/bin/convertmc", line 8, in <module>
    sys.exit(main())
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/run.py", line 153, in main
    status = convertfromlist(parsed_args.input,
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/input_output.py", line 176, in convertfromlist
    estimator = fromfilelist(filelist, error, nan)
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/input_output.py", line 77, in fromfilelist
    result = fromfile(input_file_list[0])
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/input_output.py", line 55, in fromfile
    if not reader.read(estimator):  # unsuccefful read
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/readers/common.py", line 23, in read
    result = self.read_data(estimator)
  File "/home/bassler/.local/lib/python3.9/site-packages/pymchelper/readers/shieldhit/reader_bdo2016.py", line 209, in read_data
    logger.debug("Detector data : " + str(estimator.data))

```
Screenshot_2021-03-13_10-59-39

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #379

### CODE REVIEW

File: pymchelper/readers/shieldhit/reader_bdo2016.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 class SHReaderBDO2016(SHReader):
     """
     Binary format reader from version >= 0.6
```
Comment (nbassler) [inline pymchelper/readers/shieldhit/reader_bdo2016.py:17]: but only until 0.8.0

Comment (nbassler) [reply pymchelper/readers/shieldhit/reader_bdo2016.py:17]: 0.6.0 <= version < 0.8.0

**Review by nbassler — APPROVED** [review]

**Comment by nbassler** [comment]
works for me now.

File: pymchelper/readers/shieldhit/reader_bdo2016.py (hunk 1 above)
Comment (grzanka) [reply pymchelper/readers/shieldhit/reader_bdo2016.py:17]: Not really, if old `detect.dat` format is used, then we still end up with bdo2016 file format even in SH12A v0.9.0

**Pull request merged**